content/systems/filecoin_nodes/repository/ipldstore/_index.md

```diff
@@ -24,13 +33,11 @@ Applications interact with IPLD via the Data Model, and IPLD handles marshalling
 
 ## IPLD in Filecoin
 
-On the Filecoin network, IPLD is used in two ways:
-
-- All system datastructures are stored in [IPLD](https://ipld.io) format, a data format akin to JSON but designed for storage, retrieval and traversal of hash-linked data DAGs.
-- Files and data stored on the Filecoin network may also be stored in IPLD format. While this is not required, it offers the benefit of supporting [selectors](https://github.com/ipld/specs/blob/master/selectors/selectors.md) to retrieve a smaller subset of the total stored data, as opposed to inefficiently downloading the data set entirely.
-
-IPLD provides a consistent and coherent abstraction above data that allows Filecoin to build and interact with complex, multi-block data structures, such as HAMT and Sharray. Filecoin uses the **DAG-CBOR** codec for the serialization and deserialization of its data structures and interacts with that data using the IPLD Data Model, upon which various tools are built. IPLD [Paths](https://github.com/ipld/specs/blob/master/data-model-layer/paths.md) are also used to address specific nodes within a linked data structure.
+IPLD is used in two ways in the Filecoin network:
+- All system datastructures are stored using DAG-CBOR ( an IPLD codec). DAG-CBOR is a more strict subset of CBOR with a predefined tagging scheme, designed for storage, retrieval and traversal of hash-linked data DAGs.
```

Comment:
extra space after (.

would add to this description that DAG-CBOR also guarantees determinism which CBOR doesn’t handle on its own.